style.css

```diff
@@ -0,0 +1,325 @@
+    font-size: 3rem;
+    margin: 50px 10px;
+}
+
+h3 {
+    color: #000000;
+    font-size: 22px;
+    margin: 10px 0 10px 16px;
+}
+
+h4 {
+    color: #000000;
+}
+
+p {
+    color: #000000;
+}
+
+img {
+    width: 100%;
+    height: 200px;
+    border-radius: 12px;
+    object-fit: cover;
+}
+
+ul {
+    padding: 0;
+}
+
+li {
+    list-style: none;
+}
+
+a {
+    text-decoration: none;
+    color: initial;
+}
+
+/* ======================================
+favorite recipes button
+======================================= */
+
+.favorites {
+    color: #555;
+    text-align: center;
+    margin-left: 10px;
+    margin-bottom: 50px;
+    background-color: lightpink;
+    width: 150px;
+    padding: 10px;
+    border-radius: 10px;
+    cursor: pointer;
+}
+
+.favorites:hover {
+    border: 2px solid #0018a4;
+    box-shadow: 0 0 8px 2px rgba(0, 0, 0, 0.1);
+}
+
+/* ======================================
+search form
+======================================= */
+
+.search-form {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 50%;
+    margin-left: 10px;
+    margin-bottom: 20px;
+    overflow: hidden;
+    border: 1px solid #777777;
+    border-radius: 50px;
+}
+
+.search-form:hover {
+    border: 2px solid #777777;
+}
+
+#text-input {
+    width: 100%;
+    height: 45px;
+    padding: 5px 15px;
+    border: none;
+    box-sizing: border-box;
+    font-size: 1em;
+    outline: none;
+}
+
+#text-input::placeholder{
+    font-size: 1em;
+    color: #777777;
+}
+
+.search-button {
+    background-color: skyblue;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 50px;
+    height: 45px;
+    border: none;
+    cursor: pointer;
+}
+
+.search-emoji {
+    font-size: 1.5rem;
+}
+
+/* ======================================
+style for the buttons
+======================================= */
+
+.button-container {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 20px;
+    gap: 30px;
+}
+
+.filter-container,
+.sort-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.filter,
+.sort,
+.random {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+}
+
+.filter-btn {
+    text-align: center;
+    background-color: #ccffe2;
+    color: #0018a4;
+    font-size: 18px;
+    border: none;
+    width: auto;
+    height: 40px;
+    border-radius: 50px;
+    padding: 8px 16px;
+    margin: 5px;
+    cursor: pointer;
+}
+
+.sort-btn {
+    text-align: center;
+    background-color: #ffecea;
+    color: #0018a4;
+    font-size: 18px;
+    border: none;
+    width: auto;
+    height: 40px;
+    border-radius: 50px;
+    padding: 8px 16px;
+    margin: 5px;
+    cursor: pointer;
+}
+
+.random-btn {
+    text-align: center;
+    background-color: beige;
+    color: #0018a4;
+    font-size: 18px;
+    border: none;
+    width: auto;
+    height: 40px;
+    border-radius: 50px;
+    padding: 8px 16px;
+    margin: 5px;
+    cursor: pointer;
```

Comment:
it looks like the different buttons share most ofd the styling. You could have a general button class that takes care of the base styling and then just style the random-btn, sort-btn etc with the styles that is unique that that specific button. To avoid repeating yourself. (Keep it DRY)